geode-docs/rest_apps/setup_config.html.md.erb

```diff
@@ -29,162 +31,233 @@ All Geode REST interface classes and required JAR files are distributed as a WAR
 
 where _install-dir_ is the server installation directory and _n.n.n_ is a version number.
 
-To enable the developer REST API service in Apache Geode, set the `start-dev-rest-api` Geode property to `true` when starting a data node using either `gfsh` or the ServerLauncher API. Setting this property to true on a data node will start up an embedded Jetty server and deploy the REST developer API WAR file.
+- [Enabling the REST API](#setup_config_enabling_rest)
+- [Starting the REST API Service](#setup_config_starting_rest)
+- [Implementing Authentication for the REST API](#setup_config_implementing_auth)
+- [Programmatic Startup](#setup_config_implementing_auth)
+
+# <a id="setup_config_enabling_rest" class="no-quick-link"></a>Enabling the REST API
+
+The REST API service for application development runs only on data nodes; you cannot run the service on a locator.
+
+To enable the Developer REST API service on a given server, set the `start-dev-rest-api` property
+to `true` when starting the data node to start an embedded Jetty server and deploy the Developer REST
+API WAR file on that node. Use either the `gfsh start server` command or the ServerLauncher API to enable this property.
```

Comment:
To help me parse what is getting started and where with the property, can you add a comma or reword the sentence?